app/src/main/res/values-v23/themes.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="AppTheme.NoActionBar.Splash" parent="AppTheme.NoActionBar">
```

Comment:
This is not necessary.
The code I was trying transparent view.
I will remove this!